src/main/scala/org/antipathy/scoozie/builder/MonadBuilder.scala

```diff
@@ -0,0 +1,37 @@
+package org.antipathy.scoozie.builder
+
+/**
+  * Class for accessing values contained in Monads
+  */
+private[scoozie] object MonadBuilder {
+  import scala.util.{Failure, Success, Try}
+
+  /**
+    * Attempt the passed in operation and raise an error if failure occurs
+    *
+    * @param operation the operation to attempt
+    * @param errorFunction the error function
+    * @tparam T the Type of object returned
+    * @return the result of a successful operation
+    */
+  def tryOperation[T](operation: () => T)(errorFunction: String => RuntimeException): T =
+    Try {
+      operation()
+    } match {
+      case Success(value)     => value
+      case Failure(exception) => throw errorFunction(exception.getMessage)
+    }
+
+  /**
+    * Attempt the passed in operation and return the value of the optional result
+    * @param operation the operation to try
+    * @param errorFunction the error to raise when `None` is returned
+    * @tparam T The type of the returned object
+    * @return the value of the optional type
+    */
+  def getOrException[T](operation: () => Option[T])(errorFunction: () => RuntimeException): T =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)